.github/workflows/tests.yml

```diff
@@ -12,15 +12,6 @@ jobs:
       fail-fast: false
       matrix:
         php:
-          - "5.3"
-          - "5.4"
-          - "5.5"
-          - "5.6"
-          - "7.0"
-          - "7.1"
-          - "7.2"
-          - "7.3"
-          - "7.4"
           - "8.0"
```

Comment:
Something funny happens, PHPUnit shows progress until 89%:
- https://github.com/glensc/php-zf1s/runs/1685017909?check_suite_focus=true

but the build is success! this is not good.

Reply:
I've ended up running same tests locally via docker, and there phpunit stops at 90%

Reply:
@falkenhawk what you think of disabling 8.0 for github actions, I ended up cherry-picking commits from the 8.0 branch, and seems it's still having issues. I don't want to make the PR itself very big, but rather make several PR's that are easier review and merge. possibly doing things in parallel, once there's basic phpunit invocation present.

or alternatively, can just ignore errors from 8.0 phpunit, merge with failing pipeline.

I propose making this PR exclude any issues with 8.0 (deal them later), either by removing 8.0 it consciously ignoring errors from 8.0 job.

ps: github actions does not have "allow_failure" equivalent yet. I've been tracking this issue for it: https://github.com/actions/toolkit/issues/399.

your input needed how to proceed.

Reply:
my next PR plans(after this PR) are (from https://github.com/zf1s/zf1/pull/34):
- add locale tests and locale fixes
- add services, mysql, pgsql

Reply:
wow I greatly appreciate your dedication.
Let's skip 8.0 for now and ditch any compatibility fixes needed for 8.0 here and deal with that in a separate PR, possibly branched off the 8.0 compatibility patch branch.

Having the whole pipeline running for php 5.3-7.4 on GHA would be awesome, as the first step. I asked travis for free OSS credits 2 weeks ago - because the jobs are currently not running at all due to depleted credits - and still no answer 🤷‍♂️

Reply:
Oh, didn't know Travis OSS credits have been limited, I'm not following the news apparently.

Reply:
Seems that enabling locales broken sql tests:
- https://github.com/zf1s/zf1/pull/34#issuecomment-739904178

so, need to enable sql testing next, then locales testing